### PR TITLE
Adds a series of ridiculous weapons to the wizard spellbook

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Mjolnir- Thor's hammer. Hit someone (or throw it at their face) to send them flying in a shower of sparks. 

Supermatter Sword - Literally a super matter shard in a sword, you dust people you hit with it. However it pulses rads and kills the user fairly quickly. Coded by @Incoming5643 

Veil Render -When used creates classic (small) nar-sie, a medium sized singularity that can't grow.

Singularity Hammer - Hitting a target with this will suck the whole room towards the point of impact because gravity magic

Trigger Multiverse War - Gives the wizard, and the entire crew, multiverse swords and the objective to hijack the shuttle (with their clones). Automatically calls the shuttle on purchase. Costs 8 points.

### CODE REVIEW

**Comment by callanrocks** [comment]
Literally anyone that criticizes this is a fascist.

**Comment by SweaterKittens** [comment]
For those of us unfamiliar with every one of these, and since they're not in the wiki, any chance you could do a one-sentence blurb about what each one does? It says it in files changed but those aren't very descriptive.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -350,6 +350,34 @@
...
+/datum/spellbook_entry/item/veil_render
+	name = "Veil Render"
+	desc = "A sword that can cut through reality itself, leading to mass destruction. Remember to run after you use it!"
+	item_path = /obj/item/weapon/veilrender
+	log_name = "VR"
+	category = "Weapons"
+
+/datum/spellbook_entry/item/mjolnir
+	name = "Mjolnir"
+	desc = "A mighty hammer on loan from Thor, God of Thunder. It crackles with barely contained power."
```
Comment (Dorsisdwarf) [inline code/game/gamemodes/wizard/spellbook.dm]: Inverted commas around on loan please

**Comment by Dorsisdwarf** [comment]
Multiverse War needs to do something permanent to the shuttle consoles, or latejoiners/dickbags will recall it.

**Comment by GunHog** [comment]
<b>FOR CHAOS!!</b>

**Comment by duncathan** [comment]
>singularity hammer

Oh good god that's terrifying
:+1:

**Comment by Niknakflak** [comment]
I thought you were done with shitty as fuck meme PRs

**Comment by Dorsisdwarf** [comment]
:sake:nak

**Comment by Saegrimr** [comment]
I really want to see someone pair the singularity hammer with timestop.

File: code/game/objects/items/weapons/singularityhammer.dm [hunk 2]
```diff
@@ -71,14 +71,14 @@
...
-	name = "Mjollnir"
+	name = "Mjolnir"
 	desc = "A weapon worthy of a god, able to strike with the force of a lightning bolt. It crackles with barely contained energy."
 	icon_state = "mjollnir0"
 	flags = CONDUCT
 	slot_flags = SLOT_BACK
 	force = 5
 	force_unwielded = 5
-	force_wielded = 20
+	force_wielded = 25
```
Comment (Anonus) [inline code/game/objects/items/weapons/singularityhammer.dm]: Are you SURE about this, it's already a 2-hit crit on unarmored targets.

**Comment by Anonus** [comment]
Are you sure the supermatter sword is a good idea, skeletons are immune to radiation and can buy the recall spell

**Comment by Incoming5643** [comment]
I had thought of maybe adding the supermatter sword to the spellbook, but at a VERY prohibitive spell point cost since if you manage to defeat the radiation hazard it's basically a cooldownless ei nath that also stuns and kills in a 10 square radius every time you smack a wall with it.

Also practical problem with sword war: It will drain the pool of ghosts nearly instantly, meaning at most people will usually only manage one clone unless they frantically try to create a new ghost ever time they manage to kill someone else.

**Comment by Robustin** [comment]
Only one has me excessively concerned is the summon nar-sie one. Does nar sie move around? Create harvesters?

**Comment by Aloraydrel** [comment]
Absolute Mad Man

**Comment by Anonus** [comment]
The supermatter sword should probably cost 5-7 points, so you have to expend most points on getting the immunity/defense against it.

**Comment by Anonus** [comment]
Also, on testing it's worth noting you have to throw it to cause the rad effects without hitting a mob, and recalling it while it's throwing will cause it to hit you and you will die a horrible, instant dead.

**Comment by KorPhaeron** [comment]
@Incoming5643

Maybe the super matter should stay admin only then to prevent the 5 minute wizard rounds that would result when they blew all their points on it.

Fucking rad proof skeletons.

**Comment by KorPhaeron** [comment]
@Robustin

It moves. It doesnt make harvesters. Its a mini singulo that cant grow.

**Comment by Zilenan91** [comment]
@kor what would stop the mini nar-sie from immediately gibbing the wizard?

**Comment by PKPenguin321** [comment]
holy shit

**Comment by KorPhaeron** [comment]
@Zilenan91 

Theres a delay between using the sword and  the singulo spawning

**Comment by duncathan** [comment]
Add the veal render for one spell point :^)

**Comment by KorPhaeron** [comment]
Dont make jokes like that I might buff the honkrender to clownbomb levels and add it

**Comment by Zilenan91** [comment]
What does the veal render do actually? I found it on mining one time but never used it.

**Comment by duncathan** [comment]
Spawns a FUCKTON of cows
That thing is responsible for one of my best rounds ever

**Comment by Nanotransistor** [comment]
Mjolnir + Instant Summons = YES

**Comment by Lumipharon** [comment]
>multicuck sword

Please no.

**Comment by duncathan** [comment]
Here we go boys

**Comment by PKPenguin321** [comment]
out of all the prs to be looked at by duncathan this was the one i thought was least likely to get merged lol

**Comment by Incoming5643** [comment]
OH GOD NO

**Comment by KorPhaeron** [comment]
I'll get the removal PR for the super matter sword ready dont worry

**Comment by GunHog** [comment]
At least give it a week.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 3]
```diff
@@ -350,6 +350,34 @@
 	limit = 3
 	category = "Assistance"
 
+/datum/spellbook_entry/item/super_matter
+	name = "Super Matter Sword"
```
Comment (Incoming5643) [inline code/game/gamemodes/wizard/spellbook.dm]: (also it's supermatter)

**Pull request merged**